CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java

```diff
@@ -152,6 +163,20 @@ private static String getAllTestCaseScripts(String dir) {
 		return "find " + dir + " -name \"*.class\" -type f -print";
 	}
 	
+	private boolean isValidTestMethod(String className){
+		boolean ret = false;
+		if(className == null || className.length() == 0) return ret;
+		
+		if (className.toUpperCase().indexOf("TEST") >= 0
+				&& className.indexOf("teSt") == -1
+				&& className.indexOf("tesT") == -1
+				&& className.indexOf("tSet") == -1
+				&& EXCLUDED_METHOD_LIST.indexOf(className) == -1) {
```

Comment:
as is:
EXCLUDED_METHOD_LIST.indexOf(className) == -1) {

to be:
EXCLUDED_METHOD_LIST.indexOf(","+className+",") == -1) {